CONTRIBUTING.md

```diff
@@ -246,6 +246,51 @@ process before running the corresponding test. This can be useful as a shortcut
 on the command line (since tests aren't always designed to be run outside the
 runtest harness).
 
+### Contributing to patch releases
+
+The process of creating a patch release is roughly as follows:
+
+1. Create a new branch (e.g. `backports-release-1.5`) against the relevant minor release
+   branch (e.g. `release-1.5`). Usually a corresponding pull request is created as well.
+
+2. Add commits, nominally from `master` (hence "backports"), to that branch.
+   See below for more information on this process.
+
+3. Run the [BaseBenchmarks.jl](https://github.com/JuliaCI/BaseBenchmarks.jl) benchmark
+   suite and [PkgEval.jl](https://github.com/JuliaCI/PkgEval.jl) package ecosystem
+   exerciser against that branch. Nominally BaseBenchmarks.jl and PkgEval.jl are
+   invoked via [NanoSolider.jl](https://github.com/JuliaCI/Nanosoldier.jl) from
+   the pull request associated with the backports branch. Fix any issues.
+
+4. Once all test and benchmark reports look good, merge the backports branch into
+   the corresponding release branch (e.g. merge `backports-release-1.5` into
+   `release-1.5`).
+
+5. Open a pull request that bumps the version of the relevant minor release to the
+   next patch version, e.g. as in [this pull request](https://github.com/JuliaLang/julia/pull/37718).
+
+6. Ping Alex Arslan to tag the patch release and update the website.
```

Comment:
I'm not sure whether this snippet should identify Alex specifically, or some group instead that distills to Alex? 😄

Reply:
New concept: "Generalized Alex."

Reply:
We could define a `@release-manager` group on Github, that contains Alex. Ever problem can be solved through introducing a level of abstraction

Reply:
> We could define a `@release-manager` group on Github, that contains Alex.

We already have the @JuliaLang/releases team, which contains Alex. Is that sufficient for this purpose?